pandas/tests/extension/json/test_json.py

```diff
@@ -143,6 +143,11 @@ def test_custom_asserts(self):
         with pytest.raises(AssertionError, match=msg):
             self.assert_frame_equal(a.to_frame(), b.to_frame())
 
+    @pytest.mark.xfail(reason="comparison method not implemented on JSONArray")
```

Comment:
is than issue number for this?

Reply:
The issue number is the first line in the method?

Reply:
in the xfail pls

Reply:
ok.